static/js/compose_actions.js

```diff
@@ -64,19 +64,35 @@ function get_focus_area(msg_type, opts) {
     }
 
     if (msg_type === "stream") {
-        return "#stream_message_recipient_stream";
+        return "#id_compose_select_stream";
     }
     return "#private_message_recipient";
 }
 
 // Export for testing
 export const _get_focus_area = get_focus_area;
 
+export function open_compose_stream_dropup() {
+    const $button = $("#id_compose_select_stream button");
```

Comment:
I'll add something to exit early if it's already open so that this function can't do a close